ods_ci/tests/Tests/200__monitor_and_manage/200__metrics/205_bias_metrics.robot

```diff
@@ -0,0 +1,199 @@
+*** Settings ***
+Documentation       TestSuite testing Bias metrics for Deployed Models
+Resource            ../../../Resources/RHOSi.resource
+Resource            ../../../Resources/OCP.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/Operators/ISVs.resource
+Resource            ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource           ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource            ../../../Resources/Page/OCPDashboard/Monitoring/Metrics.robot
+Library             OpenShiftLibrary
+Suite Setup         Bias Metrics Suite Setup
+Suite Teardown      RHOSi Teardown
+
+*** Variables ***
+${DEFAULT_MONITORING_NS}=            openshift-user-workload-monitoring
+${SETUP_MONITORING}=                 ${TRUE}
+${PRJ_TITLE}=                        model-serving-project
+${MODEL_ALPHA}=                      demo-loan-nn-onnx-alpha
+${MODEL_PATH_ALPHA}=                 trusty/loan_model_alpha.onnx
+${MODEL_BETA}=                       demo-loan-nn-onnx-beta
+${MODEL_PATH_BETA}=                  trusty/loan_model_beta.onnx
+${TRUSTYAI_RESOURCEPATH}=            ods_ci/tests/Resources/Files/TrustyAI
+${TRUSTYAI_CR_FILEPATH}=             ${TRUSTYAI_RESOURCEPATH}/trustyai_crd.yaml
+${MONITORING_CONFIG_FILEPATH}=       ods_ci/tests/Resources/Files/cluster-monitoring-config.yaml
+${UWM_CONFIG_FILEPATH}=              ods_ci/tests/Resources/Files/user-workload-monitoring-config.yaml
+${aws_bucket}=                       rhods-public
+${RUNTIME_NAME}=                     Model Bias Serving Test
+${PRJ_DESCRIPTION}=                  Model Bias Project Description
+${framework_onnx}=                   onnx - 1
+
+*** Test Cases ***
+
+Verify TrustyAI Operator Installation
```

Comment:
what about having this test in ods_ci/tests/Tests/100__deploy/100__installation?